### PR TITLE
VPC IPv4 validation

### PR DESCRIPTION
**Pull request by acsbendi**

### CODE REVIEW

**Comment by terrycain** [comment]
Nice, will probably annoy people who have incorrect cidrs :wink:

@JackDanger I think this ones a good idea as is more AWS like, but will probably upset some people.

If I hear nothing in a week, i'll just merge :smile:

**Comment by acsbendi** [comment]
@terrycain Thanks :) That is a possibility for sure, but I think they would also benefit from proper validation in the long term.

**Comment by mikegrima** [comment]
@terrycain Should this be merged?

File: moto/ec2/models.py [hunk 1]
```diff
@@ -2132,6 +2137,12 @@ def get_instances(cls):
 
     def create_vpc(self, cidr_block, instance_tenancy='default', amazon_provided_ipv6_cidr_block=False):
         vpc_id = random_vpc_id()
+        try:
+            vpc_cidr_block = ipaddress.IPv4Network(six.text_type(cidr_block), strict=False)
+        except ValueError:
+            raise InvalidCIDRBlockParameterError(cidr_block)
+        if vpc_cidr_block.prefixlen < 16 or vpc_cidr_block.prefixlen > 28:
```
Comment (terrycain) [inline moto/ec2/models.py]: +1

**Comment by terrycain** [comment]
@mikegrima I'm ok with this, 2nd opinion never hurts though

**Comment by terrycain** [comment]
@acsbendi fix conflicts and ill merge later

**Comment by acsbendi** [comment]
@terrycain Done :)

**Pull request merged**